Gems/Archive/Code/Source/Clients/ArchiveSystemComponent.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+#include <Archive/ArchiveBus.h>
+
+namespace Archive
+{
+    class ArchiveSystemComponent
```

Comment:
Guessing this is another boilerplate thing that may/may not be used later

Reply:
Well the SystemCompoent will definitely be used later.

Some kind of registration will be used with a UI can be in the future.

However the TickBus boilerplate and `${gem_name}Interface` classes have been removed.